CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.2.1]
+### Fixed
```

Comment:
I think we want to move `Fixed` to be the last section so that we follow the order from https://keepachangelog.com/en/1.0.0/: Added, Removed, Fixed

Reply:
done